### ISSUE TITLE
[Bug] 

### ISSUE THREAD

**Issue by rumirinbrah** [AppFlowy-IO/AppFlowy#6800]
### Bug Description

Hey there I really like your app but I found a small issue. Home and end are not working, I mean you know how you get directly to the end of a line when you click end button? yea that, that is not working

### How to Reproduce

I'm not sure

### Expected Behavior

Home and end should work for going through lines

### Operating System

Windows

### AppFlowy Version(s)

1.0.0

**Comment by rumirinbrah** [comment]
> Hi, @rumirinbrah. Can you check if you've redefined the shortcuts in the settings? I can't reproduce the issue on Windows 11. Home/End works to navigate to the beginning/end of the sentence.

Yes the shortcuts were the default. I set the keys to numpad 1 and numpad 7 & it works. But home and end keys still dont work. Maybe its an issue on my pc only?
![Screenshot 2024-11-27 140848](https://github.com/user-attachments/assets/8036b0b7-259a-4da2-8135-978d3b0a8382)

**Comment by LucasXu0** [comment]
@rumirinbrah You can reset the default keybinding for the Home/End shorctus. Only one keybinding can be active at a time.

**Comment by rumirinbrah** [comment]
@LucasXu0 Yes I tried resetting them & its the same as before

**Comment by Cxx-mlr** [comment]
I am encountering the same issue where the Home and End keys are non-functional. As a workaround, I had to manually reassign the Home key to Numpad 7 and the End key to Numpad 1.

Default keybindings:

![image](https://github.com/user-attachments/assets/88fec8af-e4ec-4500-9e13-459cea26f059)

After customizing keybindings:

![image](https://github.com/user-attachments/assets/094035a0-5744-4e75-9b99-d8e9868faab1)

I believe the default settings should work correctly without the need for any customization.

**Operating System**

Windows

**AppFlowy Version(s)**

0.7.8

**Comment by Cxx-mlr** [comment]
Additionally, these keybindings fail to work as intended in various parts of the app. For instance, when typing in the search input, pressing `Numpad 7` doesn’t jump the cursor to the start of the line, and `Numpad 1` doesn’t move it to the end. 

These keybindings do not function properly in any input field across the entire application.

**Comment by rumirinbrah** [comment]
@Cxx-mlr yes thats exactly what I meant...I wonder if it works on some other os though
